docs/SignalProvider.md

```diff
@@ -0,0 +1,28 @@
+# SignalProvider
+
+`SignalProvider` uses Signaler to tell an app what A/B tests are active.
+
+## Usage
+
+`SignalProvider` should wrap the root element or at least the largest subtree of your app that needs A/B testing.
+
+```jsx
+React.render(
+  <SignalProvider>
+    <KillerAbTestedApp/>
+  </SignalProvider>
+);
+
+```
+
+Nothing will be rendered until signals are retrieved.
```

Comment:
The simplest way to accomplish this makes the children a function so a consumer can decide what to do with the signals if they are `undefined`.